### ISSUE TITLE
Feature Request: Azure Cloud Shell option to select PowerShell instead of bash

### ISSUE THREAD

**Issue by AlanMcBee** [microsoft/terminal#2266]
# Description of the new feature/enhancement

Connecting to Azure Cloud Shell using Terminal 0.3 will automatically launch the bash cloud shell provider. Personally, I prefer PowerShell.

I am able to launch PowerShell from bash by using `pwsh`, but there is a bug: when running PowerShell Core (launched from bash) using the Azure Cloud Shell, the cursor position gets repositioned to 0,0 on every new command, and the output is overlaid on the previous output (gets very messy, hard to read).

While it is probably a good idea to address that bug, this request is just to add or modify a profile to support PowerShell Core. (I'm going to create a new issue for that bug and update this.)
UPDATED: Here is the bug report: https://github.com/microsoft/terminal/issues/2267

# Proposed technical implementation details (optional)

I thought it could just be possible to have Terminal go directly to PowerShell (preferably PowerShell Core) as an *option* for the Azure Cloud Shell profile settings, or else as a _whole new profile_.

As one of the profiles listed for a new tab in Terminal, I'd be okay with just one Azure Cloud Shell profile, where I would need to modify the settings.json to configure that profile to select the PowerShell Core option instead of the default bash option, or else there could be two Azure Cloud Shell profiles, one for bash and one for PowerShell Core.

**Comment by bitcrazed** [comment]
Currently blocked on #1883 as per comments here: https://github.com/microsoft/terminal/blob/886d018bb4032018ab69829705ea991a486c871e/src/cascadia/TerminalConnection/AzureConnection.cpp#L664-L667

**Comment by Ayanmullick** [comment]
I tried to run PowerShell from bash and it fails.
![image](https://media.discordapp.net/attachments/575331370264428584/682396580141334532/terminal.gif)

**Comment by DHowett-MSFT** [comment]
@Ayanmullick Lucky you! You hit a bug that I just filed upstream on PowerShell: https://github.com/PowerShell/PSReadLine/issues/1376
